db/migrate/20230118101945_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] The like should have an `author_id`, which means the reference should refer to `:author`, not `:user`. You can point Rails to the user table using the `to_table` option.